proto/sample_v2_round_trip.json

```diff
@@ -1115,7 +1115,7 @@
                         "name": "WordPress"
                     }
                 ],
-                "summary": {},
```

Comment:
@exterkamp this is expected now, right? I've lost track of all the summary fixes and what's real and what's just tests 😆

Reply:
No.  @brendankenny can get his proto to output `{}` instead of `null`, but I can't get mine to, and I guess @mattzeunert can't either.  `{}` should be considered canonical imo, and we need to source out this difference.

Reply:
make sure you're using the latest protobuf compiler.

Reply:
I'm on protoc v=3.7.1

I just recompiled from source.  I'm going to try to uninstall and reinstall.

Reply:
Ran `brew upgrade protobuf` to upgrade to 3.7.1 and it works fine for me now.

Reply:
> No. @brendankenny can get his proto to output {} instead of null, but I can't get mine to, and I guess @mattzeunert can't either

I guess @exterkamp now stands alone with his buggy protoc :)